common/djangoapps/student/views.py

```diff
@@ -328,7 +329,8 @@ def _cert_info(user, course, cert_status):
         'status': status,
         'show_download_url': status == 'ready',
         'show_disabled_download_button': status == 'generating',
-        'mode': cert_status.get('mode', None)
+        'mode': cert_status.get('mode', None),
+        'linked_in_url': False
```

Comment:
I think `None` would be clearer than `False`.  We generally use `None` as a placeholder when a value is not being provided.